modules/oadp-installation-issues.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+[id="oadp-installation-issues_{context}"]
+= Installation issues
+
+You might encounter issues caused by using invalid directories or incorrect credentials when you install the Data Protection Application.
+
+[id="oadp-backup-location-contains-invalid-directories_{context}"]
+== Backup storage contains invalid directories
+
+The `Velero` pod log displays the error message, `Backup storage contains invalid top-level directories`.
+
+.Cause
+
+The object storage contains top-level directories that are not Velero directories.
+
+.Solution
+
+If the object storage is not dedicated to Velero, you must specify a prefix for the bucket by setting the `spec.backupLocations.velero.objectStorage.prefix` parameter in the `DataProtectionApplication` manifest.
+
+[id="oadp-incorrect-aws-credentials_{context}"]
+== Incorrect AWS credentials
+
+The `oadp-aws-registry` pod log displays the error message, `InvalidAccessKeyId: The AWS Access Key Id you provided does not exist in our records.`
+
+The `Velero` pod log displays the error message, `NoCredentialProviders: no valid providers in chain`.
+
+.Cause
+
+The `credentials-velero` file used to create the `Secret` object is incorrectly formatted.
+
+.Solution
+
+Ensure that the `credentials-velero` file is correctly formatted, as in the following example:
+
+.Example `credentials-velero` file
+----
+[default] <1>
+aws_access_key_id=AKIAIOSFODNN7EXAMPLE <2>
+aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+----
+<1> AWS default profile.
+<2> Do not enclose the values with quotation marks (`"`, `'`).
+
+[id="oadp-incorrect-azure-credentials_{context}"]
+== Incorrect Azure credentials
+
+The `Velero` pod log displays one of the following error messages:
+
+* `unable to get all required environment variables: the following keys do not have values: AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, AZURE_SUBSCRIPTION_ID`
```

Comment:
Not sure if this point is linked to the cause below. Need to double check

Reply:
This section came from a Velero issue. I don't want to make extra work for you so I will take it out.